src/+otp/+vanderpol/+presets/ExtendedPrecision.m

```diff
@@ -0,0 +1,28 @@
+classdef ExtendedPrecision < otp.vanderpol.VanderpolProblem
+    methods
+        function obj = ExtendedPrecision(epsilon)
+            if nargin < 1
+                epsilon = sym(10^-6);
```

Comment:
maybe use "1e-6"? I would think that is more standard.

Reply:
actually I think we would want to do "sym(10)^(-6)" to not lose any precision.

Reply:
I think `vpa(1e-6)` is what we want.

Reply:
Doesn't matter in this case. 
sym(10^-6) - vpa(1e-6) = 0.0

Reply:
It does matter. Vpa has finite precision and behaves differently than sym.  I don't think sym is what we want

Reply:
So right now, if the user provides epsilon as  ``sym(1)/100``  the initial condition would be symbolic rational numbers. If it's set is vpa(1e-2, 30) then Y0 is type symbolic with 30 digits. If it's double output is double too. I think that's the best that can be done.  Question is what should be the default case. I vote for symbolic rational numbers, cuz the user can always truncate them later.